### PR TITLE
feat: make icons respond to user theme selection

### PR DESCRIPTION
**Pull request by adamnel**
#69 SVG paths now show/hide based on the class applied to the body tag by theme service:

image

### CODE REVIEW

File: src/docs-ui/app/app.component.scss [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// Icons - Swap based on app theme
```
Comment (adamnel) [inline src/docs-ui/app/app.component.scss]: Is there a better place to put this? Tried for a while to make it work in _overrides but no joy.

Comment (emoralesb05) [reply src/docs-ui/app/app.component.scss]: Yeah, we need to put it in one of those files to be able to distribute it. 

_overrides didnt work?

Comment (adamnel) [reply src/docs-ui/app/app.component.scss]: Nope, has to do with the cascade of the component SCSS I think - :host works in any component theme file in the chain, but not more general global css like _overrides or theme.scss.

Looked into view encapsulation for this but that also seems to only apply to components.

Comment (adamnel) [reply src/docs-ui/app/app.component.scss]: Nevermind, got it

File: src/lib/theme/_overrides.scss [hunk 2]
```diff
@@ -89,6 +89,28 @@ $shape-radius: 8px;
     }
   }
 
+  // Icons - Swap based on app theme
+  .light-theme {
```
Comment (emoralesb05) [inline src/lib/theme/_overrides.scss]: I would remove this wrapper class and have it be the default, since this class sometimes is not added.

Comment (adamnel) [reply src/lib/theme/_overrides.scss]: It's already that way based on the SVG internal CSS but yeah can't hurt

**Comment by owilliams320** [comment]
LGTM

**Pull request merged**